README.md

```diff
@@ -82,10 +82,10 @@ $ minikube start --vm-driver=xhyve
 **Deploy FaaS-netes and the API Gateway**
 
 ```
-$ kubectl apply -f ./faas.yml,monitoring.yml
+$ kubectl apply -f ./faas.yml,monitoring-config.yml,monitoring.yml
```

Comment:
Can you add the `rbac.yml` here and get rid of the optional section. RBAC isn't optional now surely?

Reply:
I think we should make that change in a upcoming PR instead. I feel that PRs should only contain singular feature changes. That way things are easier to track. Especially if things break, it becomes much easier to revert. I believe there's also a discussion in the issues section about the changes in k8s 1.8 with the rbac api version.

Reply:
Ok, fair enough.